docs/devcontainers/devcontainer-cli.md

```diff
@@ -1,13 +1,13 @@
 ---
 Order: 14
 Area: devcontainers
-TOCTitle: devcontainer CLI
+TOCTitle: Dev Containers CLI
 PageTitle: Installing and working with the devcontainer CLI
 ContentId: 8946213d-716e-41ca-955f-944a41c70353
 MetaDescription: Documentation on using the development container (dev container) command-line interface
 DateApproved: 10/6/2022
 ---
-# Development container CLI
+# Dev Containers CLI
```

Comment:
We use singular container when referring to the CLI (it's separate from the Dev Containers extension) https://github.com/devcontainers/cli

```suggestion
# Dev Container CLI
```